app/src/main/java/com/greenfox/opal/gitinder/model/Profile.java

```diff
@@ -0,0 +1,29 @@
+package com.greenfox.opal.gitinder.model;
+
+import java.util.List;
+
+public class Profile {
```

Comment:
@Garlyle these models supposed to be responses from the server, hence these should be there